daemon/algod/api/algod.oas3.yml

```diff
@@ -2448,11 +2448,12 @@
             }
           },
           {
-            "description": "Number of box names to return. If max is not set, or max == 0, returns all box-names.",
+            "description": "A box name",
```

Comment:
@algochoi Minor:  When sending a value for `name`, how will the API user know to apply goal argument encoding?

I'm thinking either the highlighted description or route description can specify the expected encoding.

Reply:
@algochoi Looks reasonable.   A few thoughts:

* Unless you're opposed, I'd phrase as "goal _app call_ arg encoding" (or similar) to qualify which subsection of goal we're referencing.
* I'm wondering how an API user discovers available encodings.  Is there an existing doc that can be linked to show available encodings? 
  * Minimally, I'm imagining we can reference `--app-arg` in  https://developer.algorand.org/docs/clis/goal/app/call/ to help the user discover supported formats.

Reply:
> I'd phrase as "goal app call arg encoding" (or similar) to qualify which subsection of goal we're referencing.

This sounds good

> Minimally, I'm imagining we can reference --app-arg in https://developer.algorand.org/docs/clis/goal/app/call/ to help the user discover supported formats.

I'm not opposed to it, but linking a static page in the API sounds like it could be contentious. At the same time, I'm not really sure how else we can improve discoverability.

Reply:
> Alternatively, I propose extracting the usage string into a shared variable (or constant) that's referenced in both places.

Suppose something like:

```
const AppCallBytesHelpString string = "For ints, use the form 'int:1234'. For raw bytes, use the form 'b64:A=='. For printable strings, use the form 'str:hello'. For addresses, use the form 'addr:XYZ...'."
```

I'm not quite sure how we would export it into the OA2 spec easily. Maybe we should just copy that part over to the spec instead?

Reply:
@algochoi Sorry, I made a nonsensical proposal (Go const vs JSON file).  Agree with duplicating inline.